iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/RbacAzureResourceCheck.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.checks;
+
+import org.sonar.check.Rule;
+import org.sonar.iac.arm.checkdsl.ContextualResource;
+
+import static org.sonar.iac.arm.checks.utils.CheckUtils.isFalse;
+
+@Rule(key = "S6383")
+public class RbacAzureResourceCheck extends AbstractArmResourceCheck {
```

Comment:
I would suggest naming this check `RoleBasedAccessControlCheck`. This is also the name of the check which is implemented for [Terraform](https://github.com/SonarSource/sonar-iac/blob/dda5f3a5cf7514fb6e5f75be38c2a16feabf51fd/iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/azure/RoleBasedAccessControlCheck.java#L33).